itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -232,8 +232,6 @@ private Tuple replDumpDb(String dbName, String fromReplID, String toReplID, Stri
   }
 
   private void loadAndVerify(String replDbName, String dumpLocation, String lastReplId) throws IOException {
-    run("EXPLAIN REPL LOAD " + replDbName + " FROM '" + dumpLocation + "'", driverMirror);
```

Comment:
why did we remove the explain statements ? is this to speed up tests ?

Reply:
It is removed for 2 reasons. After fixing the ALTER_PARTITION issue, the length of DAG increased which caused stack overflow with EXPLAIN query on REPL LOAD (Created another bug to track this issue). The 2nd reason is EXPLAIN REPL LOAD is needed only for debugging and has no benefit with automated run and hence removed to increase the speed.